### PR TITLE
Add Spark Thrift Server V2.4.7 helm charts

### PR DESCRIPTION
**Pull request by akshayharyani**

### CODE REVIEW

File: charts/spark-ts-2.4.7/spark-ts-chart/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{{ $componentName := include "spark-ts-chart.serviceAccountName" . }}
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $componentName }}
```
Comment (AlexNavara) [inline charts/spark-ts-2.4.7/spark-ts-chart/templates/serviceaccount.yaml:6]: Maybe we should put here "spark-ts-chart.serviceAccountName"?

**Review by AlexNavara — CHANGES_REQUESTED** [review]

**Comment by apurwaj2** [comment]
In the common.sh script of PKD, it expects POD_SERVICE_ACCOUNT ==  hpe-${POD_NAMESPACE}, to set ADMIN_SECRET to 'user' otherwise it is set to denied if a different service account. I don't know whether it affects thrift-server. It was an issue in case of hivemetastore and so I did not create a new service-account.  In case, you've already taken a look at this and you don't need to stick to using the hpe-${POD_NAMESPACE}, ignore the comment.

File: charts/spark-ts-2.4.7/spark-ts-chart/templates/serviceaccount.yaml (hunk 1 above)
Comment (akshayharyani) [reply charts/spark-ts-2.4.7/spark-ts-chart/templates/serviceaccount.yaml:6]: I am copying  `spark-ts-chart.serviceAccountName` to a local variable called `componentName` and using it here and everywhere in this yaml. So the actual value will be same as of using `spark-ts-chart.serviceAccountName`

Comment (AlexNavara) [reply charts/spark-ts-2.4.7/spark-ts-chart/templates/serviceaccount.yaml:6]: Ok, thanks.

**Review by AlexNavara — APPROVED** [review]
lgtm

**Review by apurwaj2 — APPROVED** [review]
lgtm

**Review by AlexNavara — APPROVED** [review]

**Pull request merged**